### PR TITLE
Added requisite for new numbers request.

### PR DESCRIPTION
**Pull request by ErsancMS**
Added requisite to include area code, country or region for new numbers request

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ If you need help with getting or transferring numbers to us but need help please
 
   - Your Organization ID (Tenant ID) or active domains.
   - What types of numbers you need help with (User/Subscriber, Service/Toll, or Toll-Free).
+    - Include the Area Code(s)/Country/Region from which you would like telephone numbers from.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
    - Include the Area Code(s)/Country/Region from which you would like telephone numbers.
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ErsancMS** [comment]
@tonysmit could you help us push this forward?

**Comment by ErsancMS** [comment]
@tonysmit ping

**Pull request merged**